packages/camera/camera_android/README.md

```diff
@@ -1,24 +1,26 @@
 # camera\_android
 
-The Android implementation of [`camera`][1].
-
-*Note*: [`camera_android_camerax`][3] will become the default implementation of
-`camera` on Android by May 2024, so **we strongly encourage you to opt into it**
-by using [these instructions][4]. If any [limitations][5] of `camera_android_camerax`
-prevent you from using it or if you run into any problems, please report these
-issues under [`flutter/flutter`][5] with `[camerax]` in the title.
+An Android implementation of [`camera`][1] built with the [Camera2 library][4].
 
 ## Usage
 
-This package is [endorsed][2], which means you can simply use `camera`
-normally. This package will be automatically included in your app when you do,
-so you do not need to add it to your `pubspec.yaml`.
+As of `camera: ^0.11.0`, to use this plugin instead of [`camera_android_camerax`][3],
+run
+
+```sh
+$ flutter pub add camera_android
+```
+
+If using `camera: <0.11.0` and wish to use this plugin instead of [`camera_android_camerax`][3],
+you can simply use `camera` normally because it is [endorsed][2]. This package will automatically
+be included in your app when you do so, so you do not need to add it to your `pubspec.yaml`.
```

Comment:
Let's delete this whole paragraph; I don't think we need to document how older versions of the app-facing plugin work. (And doing the above is largely harmless for old versions anyway.)